src/test/java/Homework17.java

```diff
@@ -0,0 +1,23 @@
+import org.openqa.selenium.By;
+import org.openqa.selenium.WebElement;
+import org.openqa.selenium.interactions.Actions;
+import org.testng.annotations.Test;
+import java.time.Duration;
+
+public class Homework17 extends BaseTest{
+
+    @Test
+    public void addSongToPlayList(){
+        openLoginUrl();
+        login("demo@class.com","te$t$tudent");
+        myDriver.manage().timeouts().implicitlyWait(Duration.ofSeconds(5));
+        pickSong();
```

Comment:
Please finish the homework 17!

Validate that there is a song added.

For example  `Assert.assertTrue(addedSong.isDisplayed());`

![image](https://user-images.githubusercontent.com/104761482/216827158-1a9acb92-f3ae-484d-9598-5b61fc063dfc.png)

![image](https://user-images.githubusercontent.com/104761482/216827190-8aee2e3f-4384-45d1-8348-f69a91fb1861.png)